### PR TITLE
don't run CSVv2 b-tagger in JMECustomNanoAOD for post-LS2 eras

### PR DESCRIPTION
**Pull request by missirol**
#### PR description:

This PR is the minimal change to fix wf `11634.15`, required to build the next pre-release (see https://github.com/cms-sw/cmssw/pull/40485#issuecomment-1383689283).

Further, future, improvements are left to JME/BTV experts.

#### PR validation:

Wf `11634.15` passes locally.

#### If this PR is a backport, please specify the original PR and why you need to backport that PR. If this PR will be backported, please specify to which release cycle the backport is meant for:

N/A

### CODE REVIEW

**Comment by missirol** [comment]
test parameters:
  - workflows = 11634.15

**Comment by missirol** [comment]
please test

File: PhysicsTools/NanoAOD/python/custom_jme_cff.py [hunk 1]
```diff
@@ -27,7 +27,9 @@
   'pfDeepFlavourJetTags:probc','pfDeepFlavourJetTags:probuds','pfDeepFlavourJetTags:probg'
 ]
 from RecoBTag.ONNXRuntime.pfParticleNetAK4_cff import _pfParticleNetAK4JetTagsAll
-bTagDiscriminatorsForAK4 = bTagCSVV2+bTagDeepCSV+bTagDeepJet+_pfParticleNetAK4JetTagsAll
+bTagDiscriminatorsForAK4 = cms.PSet(foo = cms.vstring(bTagCSVV2+bTagDeepCSV+bTagDeepJet+_pfParticleNetAK4JetTagsAll))
```
Comment (missirol) [inline PhysicsTools/NanoAOD/python/custom_jme_cff.py:30]: I wouldn't be suprised if there was a better way to do this (`.toModify/toReplaceWith` accept only certain types, and I didn't find a better way).

**Comment by tvami** [comment]
@cms-sw/xpog-l2 I tested this PR together with 40485 where `nano` was enable, please find the numbers in
https://github.com/cms-sw/cmssw/pull/40485#issuecomment-1384277391

**Comment by missirol** [comment]
please close

**Pull request closed without merging**